src/styles/cards.scss

```
@@ -0,0 +1,109 @@
+.card {
+  padding-top: 32px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  transition-duration: 300ms;
+
+  &:hover.card__name {
+    color: #34568b;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-top: 40px;
+    padding: 0 16px;
+    color: $main-accent;
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin: 0;
+    margin-top: 4px;
+    padding: 0 16px;
+    color: $secondary;
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__grade {
+    margin-top: 16px;
+    padding: 0 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-review {
+      margin: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: center;
+    }
+  }
+
+  &__total {
+    margin-top: 24px;
+    padding: 0 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-price {
+      margin: 0;
+      color: $secondary;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-sum {
+      margin: 0;
+      color: $main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    margin: 16px auto;
+    padding-top: 12px;
+    width: 166px;
```

```suggestion
    width: 100%;
```
The button should have the entire width of its container.